### PR TITLE
wine: add x86_64 target with i686 support

### PR DESCRIPTION
**Pull request by tim241**

### CODE REVIEW

**Comment by tim241** [comment]
[it builds correctly](https://travis-ci.org/void-linux/void-packages/jobs/395781029)

**Comment by abenson** [comment]
It built for me. And I'm about to test with Skyrim:SE.

**Comment by abenson** [comment]
It works, but I need to work out some serious performance issues.

**Comment by abenson** [comment]
I've tried a few games and applications so far. 64-bit works fine, and 32-bit on 64-bit also works fine.

File: srcpkgs/wine/template [hunk 1]
```diff
@@ -13,8 +14,10 @@ checksum=d7505cc79c7cd8950f1a46ecfcc0ae2ecc71717ce85414846264395b1e7ac6b5
...
 
-lib32mode=full
-only_for_archs=i686
+only_for_archs="i686 x86_64"
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	lib32mode=full
+fi
 
 hostmakedepends="pkg-config flex prelink"
 makedepends="gettext-devel lcms2-devel zlib-devel ncurses-devel
```
Comment (abenson) [inline srcpkgs/wine/template]: This can be reduced to: gettext-devel lcms2-devel ncurses-devel

**Comment by abenson** [comment]
That's really good to know! I tried to get dxvk working for me but it didn't work, but I don't think it's related to the wine build. I may try again.

**Comment by abenson** [comment]
This is the error I get.
~~~
01d9:err:vulkan:get_vulkan_driver Wine was built without Vulkan support.
01d9:err:vulkan:wine_vk_init Failed to load Wine graphics driver supporting Vulkan.
01d9:err:module:attach_dlls "winevulkan.dll" failed to initialize, aborting
~~~

**Comment by abenson** [comment]
It also looks like 3.11 is out. I'm building it right now.

**Comment by tim241** [comment]
@abenson I'll add vkd3d as a package too(because that's why vulkan wasn't built-in, soon though, when they allow/merge/review this PR)

**Comment by nathanielhorn** [comment]
@abenson in the meantime you can try this script to compile vulkan with the prefix 
Https://github.com/gloriouseggroll/dxvk-tools 

I haven't used it in a while but this worked great for me before wine started adding vulkan support by default.

**Comment by tim241** [comment]
I didn't mean to do that, sorry! (Will open another PR since I can't reopen this one)

**Comment by abenson** [comment]
You should be able to reopen it! You can undelete branches on github for a short period of time

**Comment by tim241** [comment]
well I tried something stupid, so I broke it(tried to delete the branch, merge upstream, then make the branch again and hoped it'd work). I'm sorry!

**Pull request closed without merging**